controls/SV-204579.rb

```diff
@@ -11,21 +11,21 @@
     application level if multiple application sessions are using a single operating system-level network connection.
     This does not mean that the operating system terminates all sessions or network access; it only ends the inactive
     session and releases the resources associated with that session.'
-  desc 'check', 'Verify the operating system terminates all network connections associated with a communications session at the end of the session or based on inactivity.
+  desc 'check', "Verify the operating system terminates all network connections associated with a communications session at the end of the session or based on inactivity.
 
 Check the value of the system inactivity timeout with the following command:
 
 $ sudo grep -irw tmout /etc/profile /etc/bashrc /etc/profile.d
 
-etc/profile.d/tmout.sh:declare -xr TMOUT=900
+etc/profile.d/tmout.sh:declare -xr TMOUT=#{input('system_activity_timeout')}
 
 If conflicting results are returned, this is a finding.
-If "TMOUT" is not set to "900" or less to enforce session termination after inactivity, this is a finding.'
-  desc 'fix', 'Configure the operating system to terminate all network connections associated with a communications
+If "TMOUT" is not set to #{input('system_activity_timeout')} or less to enforce session termination after inactivity, this is a finding."
```

Comment:
@raziz086 @wajidraja `"TMOUT"` not escaped so it's causing a syntax error
Please run `inspec exec . -t mock://` from within the profile to see if there are additional syntax errors

Reply:
Updated

Reply:
Reviewed - thanks!